src/components/organisms/MessageBox/MessageBox.tsx

```diff
@@ -36,7 +50,7 @@ const MessageBox: React.FC = () => {
       // @ts-ignore
       notification[notificationType]({
         key: alert.id,
-        message: alert.title,
+        message: modify(alert.title),
```

Comment:
Here we probably had a problem before, I think it should've been the `alert.message` being used..